CRM/CiviAwards/Service/ApplicantManagementAwardDetailPostProcessor.php

```diff
@@ -124,8 +124,8 @@ public function processCaseTypeCustomGroupsOnUpdate(AwardDetail $awardDetail) {
    * @return bool
    *   Whether award subtype changed or not.
    */
-  private function awardSubTypeChanged(AwardDetail $awardDetail) {
-    return !empty($awardDetail->award_type) && $awardDetail->award_type != $awardDetail->oldAwardDetail->award_type;
+  private function awardSubtypeChanged(AwardDetail $awardDetail) {
```

Comment:
Do we need to change the function name here?

Reply:
Since now we have `Subtype` throughout the application, I refactored this to follow same convention. Maybe we can keep it?